pygac/tests/test_reader.py

```diff
@@ -298,6 +298,57 @@ def test_get_tle_lines(self, read_tle_file, *mocks):
+                                      27.6796875, 27.58467527, 27.48959853, 27.39453053,  2.5704025,
+                                      2.30362323],
+                                     [69.44430772, 68.80064104, 27.91910034, 27.82340242, 27.72794715,
+                                      27.6328125, 27.53805662, 27.44366144, 27.34959008,  2.53088093,
+                                      2.26729486],
+                                     [69.47408815, 68.8259859, 27.87666513, 27.78224611, 27.68795045,
+                                      27.59375, 27.49962326, 27.40557682, 27.31162435,  2.48359319,
+                                      2.21976689]])
+        self.reader.lats = np.array([[71.62830288, 71.67081539, 69.90976034, 69.89297223, 69.87616536,
+                                      69.859375, 69.84262997, 69.82593315, 69.80928089, 61.61334632,
+                                      61.466097],
+                                     [71.65222644, 71.69455292, 69.9331981, 69.91640991, 69.89960295,
+                                      69.8828125, 69.86606741, 69.84937054, 69.83271828, 61.62903602,
+                                      61.48182119],
+                                     [71.68324681, 71.7256586, 69.96444365, 69.94765637, 69.93085095,
+                                      69.9140625, 69.89731955, 69.8806245, 69.86397337, 61.65247459,
+                                      61.50526043],
+                                     [71.71489701, 71.75715746, 69.98789144, 69.97110185, 69.9542928,
+                                      69.9375, 69.92075278, 69.90405431, 69.88740114, 61.66489162,
+                                      61.51597801],
+                                     [71.7390608, 71.78108857, 70.01126173, 69.99449392, 69.97770882,
+                                      69.9609375, 69.9442054, 69.92751555, 69.91086544, 61.67769951,
+                                      61.52802445]])
+        self.reader.times = np.array([datetime.datetime(1980, 1, 3, 11, 47, 15, 469000),
+                                      datetime.datetime(1980, 1, 3, 11, 47, 15, 969000),
+                                      datetime.datetime(1980, 1, 3, 11, 47, 16, 469000),
+                                      datetime.datetime(1980, 1, 3, 11, 47, 16, 969000),
+                                      datetime.datetime(1980, 1, 3, 11, 47, 17, 469000)])
+        expected_sat_azi_0 = np.array([283.09872924, 283.12775589, 283.13951497, 283.14786413, 283.19638805])
+        expected_sat_azi_201 = np.array([272.85051989, 273.79847634, 272.04794616, 273.01363377, 274.00055397])
+        expected_sat_azi_408 = np.array([39.77021472, 39.71516966, 39.68104134, 39.60503726, 39.5403431])
+        expected_sat_elev_0 = np.array([20.94889204, 20.96041284, 20.96368521, 20.96826309, 20.95849212])
+        expected_sat_elev_204 = np.array([89.24533884, 89.22663677, 89.25079817, 89.24938043, 89.23004118])
```

Comment:
Were these expected values computed with TLE and then for the tests they are approximated without TLE?

Reply:
Yes, that is how I made theml.